mscore/palette.cpp

```diff
@@ -274,6 +274,20 @@ void Palette::contextMenuEvent(QContextMenuEvent* event)
             return;
             }
 
+      if (_isSymbolsPaletteInMasterPalette) {
+            QMenu menu;
+            QAction* copyNameMenuItem = menu.addAction(tr("Copy SMuFL Symbol Code"));
+            if (menu.exec(mapToGlobal(event->pos())) == copyNameMenuItem) {
+                  PaletteCell* cell = cellAt(i);
+                  if (cell) {
+                        QRegularExpression regex("<sym>(.+?)</sym>");
+                        QString symSmuflName = QString("<sym>%1</sym>").arg(regex.match(cell->name).captured(1));
+                        QApplication::clipboard()->setText(symSmuflName);
+                        }
+                  }
+            return;
+            }
+
       QMenu menu;
       QAction* deleteCellAction   = menu.addAction(tr("Delete"));
       QAction* contextAction = menu.addAction(tr("Properties…"));
```

Comment:
Can't we get rid of these two options in master symbols palette, where neither does anything anyway?

Reply:
For the symbols section, I removed them with this PR, and for the 'normal' sections, there is somehow no right-click menu at all (right click does for me the same as left click).

Reply:
Ah, right, I missed the `return` statement above